packages/app/src/specs/SpecsList.vue

```diff
@@ -79,17 +79,17 @@
     />
     <div
       v-if="specs.length"
-      class="mb-4 grid grid-cols-7 children:font-medium children:text-gray-800"
-      :style="`padding-right: ${scrollbarOffset}px`"
+      class="mb-4 grid grid-cols-5 children:font-medium children:text-gray-800"
+      :style="`padding-right: ${scrollbarOffset && scrollbarOffset + 20}px`"
```

Comment:
Is there a reason we don't want to include a little padding, even when scrollbars may not be present? macOS temporary scrollbars can overlay the content as-is:

![image](https://user-images.githubusercontent.com/1711637/173616964-5de7f6de-2d5a-4529-b556-3df78dfbf1bb.png)

Reply:
@tbiethman what browser does this?

Reply:
That was from Chrome on macOS, with "show scroll bars" set to automatic:

<img width="671" alt="Screen Shot 2022-06-14 at 1 07 59 PM" src="https://user-images.githubusercontent.com/1711637/173659316-23ea59f9-0e4f-4c7c-a236-596ab8de30bf.png">

Reply:
Addressed in last commit